extended-it/src/test/java/apoc/vectordb/MilvusTest.java

```diff
@@ -117,6 +117,17 @@ public void before() {
         dropAndDeleteAll(db);
     }
 
+    @Test
+    public void getInfo() {
+        testResult(db, "CALL apoc.vectordb.milvus.info($host, 'test_collection', '', $conf) ",
+                map("host", HOST, "conf", map(FIELDS_KEY, FIELDS)),
+                r -> {
+                    Map<String, Object> row = r.next();
+                    Map value = (Map) row.get("value");
+                    assertEquals(200L, value.get("code"));
+                });
+    }
```

Comment:
Metti così, per far capire che in questo modo l'utente può passare il dbName se vuole.

```suggestion
    @Test
    public void getInfo() {
        testCall(db, "CALL apoc.vectordb.milvus.info($host, 'test_collection', $conf) ",
                map("host", HOST, "conf", map()),
                r -> {
                    Map value = (Map) r.get("value");
                    assertEquals(200L, value.get("code"));
                });
    }

    @Test
    public void getInfoWithWrongDatabaseName() {
        testCall(db, "CALL apoc.vectordb.milvus.info($host, 'test_collection', $conf) ",
                map("host", HOST, "conf", map(BODY_KEY, map("dbName", "wrong"))),
                r -> {
                    // todo - assertion contenente "wrong database"
                });
    }

    @Test
    public void getInfoWithRightDatabaseName() {
        testCall(db, "CALL apoc.vectordb.milvus.info($host, 'test_collection', $conf) ",
                map("host", HOST, "conf", map(BODY_KEY, map("dbName", "default"))),
                r -> {
                    Map value = (Map) r.get("value");
                    assertEquals(200L, value.get("code"));
                });
    }
```

Reply:
PS: sicuramente non te l'ho detto, comunque quando la procedura ritorna una sola riga, puoi mettere `testCall` invece di `testResult`, che sottobanco fa direttamente il next() e quindi non devi scrivere `Map<String, Object> row = r.next();`